desktop/core/src/desktop/templates/hue.mako

```diff
@@ -295,8 +296,9 @@ ${ hueIcons.symbols() }
 </div>
 ${ commonshare() | n,unicode }
 
-% for bundle in get_hue_bundles('hue'):
-  ${ render_bundle(bundle) | n,unicode }
+% for bundle in get_hue_bundles('login' if section == 'login' else 'hue', 'LOGIN' if section == 'login' else 'DEFAULT'):
```

Comment:
Why the login related checks for hue.mako?

Reply:
reverted